backend/de.metas.business/src/main/java/de/metas/bpartner/composite/BPartnerLocationType.java

```diff
@@ -53,16 +52,12 @@
 	@JsonInclude(Include.NON_ABSENT)
 	private Optional<Boolean> shipToDefault;
 
-	@JsonInclude(Include.NON_ABSENT)
-	private Optional<Boolean> visitorsAddress;
```

Comment:
we have this in the `master` branch, so i'm a bit nervous about loosing it in here. please check if that isn't an accident

Reply:
we have it here too, but directly in `BPartnerLocation`, form my POV it doesn't make any sense to keep it here too